src/libcmis/gdrive-document.cxx

```diff
@@ -27,6 +27,7 @@
  */
 
 #include "gdrive-document.hxx"
+#include <algorithm>
```

Comment:
Also, please use `git commit --amend` or `git rebase --interactive` to make "perfect" commits (followed by `git push --force`).

Reply:
I tried too but sometimes the push is rejected on my local branch.
This time it seemed ok.